subprojects/core-api/src/main/java/org/gradle/api/artifacts/MutableVersionConstraint.java

```diff
@@ -64,29 +79,17 @@
      * This is a very soft version declaration.
      * It applies only if there is no stronger non dynamic opinion on a version for the module.
      * This term does not support dynamic versions.
+     * This clears any set rejected versions.
```

Comment:
I would move this sentence below the next one and make it a different paragraph as well.